### ISSUE TITLE
Update examples to work with latest version of code

### ISSUE THREAD

**Issue by alexdewar** [imperialCHEPI/healthgps-examples#28]
The main problem seems to be that the static linear model definition has changed, but it looks like the EBHLM model definition may also change (https://github.com/imperialCHEPI/healthgps/issues/531).

This doesn't need to be done until I've released the examples (#25). Are you happy to take this one at that point @jamesturner246? I'd be happy to have a go, but I'm not sure what realistic numbers for the new parameters would look like.

**Comment by jamesturner246** [comment]
I'll put this on hold whilst the example stuff is ongoing.

**Comment by jamesturner246** [comment]
Some extra notes: mostly KevinHall etcetera schemas are updated, so old models shouldn't need much change if any, only newer.

Also CI needs updating to use the latest schemas available for each model config.
